src/components/spaces/organisms/organelle-wrapper.tsx

```diff
@@ -286,10 +286,10 @@ export class OrganelleWrapper extends BaseComponent<OrganelleWrapperProps, Organ
     modelDef.properties = modelProperties;
 
     createModel(modelDef).then((m: any) => {
-      // appStore.boxes.get(this.props.boxId).setModel(m);
       this.model = m;
       this.model.zoomLevel = zoomLevel;
       this.model.mode = row.mode;
+      this.setState({dropperCoords: []});
```

Comment:
Removes it a little later than I'd like...any earlier and I was getting some exceptions I couldn't understand off the top of my head.